python/hsfs/version.py

```diff
@@ -14,4 +14,4 @@
 #   limitations under the License.
 #
 
-__version__ = "3.1.1rc1"
```

Comment:
You need to bump the version also on the java pom and the java utils pom

Reply:
done